### PR TITLE
MTM-49016: Change log for Tenant options API security improvement

### PR DESCRIPTION
**Pull request by kowa-c8y**

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Tenant options API security improvement
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md:3]: I think the API is called Tenant API?

Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md:3]:
```suggestion
title: Tenant API security improvement
```

Comment (kowam-c8y) [reply content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md:3]: This is exactly this API: [option](https://cumulocity.com/api/core/#operation/getOptionCollectionResource) so it's more options than tenant API but please let me know which wone is better from your perspective :) Is body text correct ?

**Review by kowa-c8y — COMMENTED** [review]
@BeateRixen

File: content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-49016
+version: 10.18.540.197
+---
+Security improvements have been implemented for credential-options on the tenant option API.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md:17]: The security for the credential options in the Tenant API has been improved.

File: content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md (hunk 1 above)
Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-18-540-197-tenant-options-api-security-improvement.md:3]: I would go with Tenant API as this is how we usually call it. "Options" is a method at least according to the documentation.

**Review by pink-mongoose — COMMENTED** [review]
It is unclear to me precisely what has been done and what the change is that the customer will experience.  Are all tenant options now encrypted?  If yes, does the customer have to do anything?

**Comment by kowam-c8y** [comment]
PR split into separate parts:
- https://github.com/SoftwareAG/c8y-docs/pull/2445
- https://github.com/SoftwareAG/c8y-docs/pull/2444
I'm closing this one.

**Pull request closed without merging**